code/game/dna/mutations/mutation_powers.dm

```diff
@@ -6,9 +6,8 @@
 	name = "No Breathing"
 	activation_messages = list("You feel no need to breathe.")
 	deactivation_messages = list("You feel the need to breathe, once more.")
-	instability = GENE_INSTABILITY_MODERATE
+	instability = GENE_INSTABILITY_MAJOR
 	traits_to_add = list(TRAIT_NOBREATH)
-	activation_prob = 25
```

Comment:
Can you just remove this var from the supertype entirely, especially if it's now unused?

Reply:
Pushed.